<launch>
	<arg name="gui" default="False" />
	<param name="robot_description" command="$(find xacro)/xacro '$(find train)/urdf/train_robot.urdf.xacro'" />
	<param name="use_gui" value="$(arg gui)"/>
	<node name="state_publisher" pkg="train" type="state_publisher" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find train)/urdf.rviz" required="true" />
</launch>
